### PR TITLE
feat: stock view controller including primitive sample data

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
In the front-end, we're working on a stock view.

This commit mainly contains:
- the next layer (Controller) that can be used to orchestrate the stock data in the frontend
- a small command line runner to have some data for demo purposes
- new Dependency ModelMapper (already dasched licence with "approved") to map between Entity and Dto classes.

@mhellmeier please review.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>3.0.2</version>
+    <version>3.0.1</version>
```
Comment (mhellmeier) [inline pom.xml:31]: Why downgrade from `3.0.2` to `3.0.1`?

File: src/main/java/org/eclipse/tractusx/puris/backend/CustomerCommandLineRunner.java [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class CustomerCommandLineRunner implements CommandLineRunner {
+
+    @Autowired
+    private ModelMapper modelMapper;
+
```
Comment (mhellmeier) [inline src/main/java/org/eclipse/tractusx/puris/backend/CustomerCommandLineRunner.java:54]: Remember to reformat the code to reduce unnecessary empty lines.

File: src/main/java/org/eclipse/tractusx/puris/backend/stock/controller/StockController.java [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    @Autowired
+    private PartnerService partnerService;
+
+
+    @Autowired
+    private ModelMapper modelMapper;
+
+    @GetMapping("materials")
+    @ResponseBody
+    public List<MaterialDto> getMaterials() {
```
Comment (mhellmeier) [inline src/main/java/org/eclipse/tractusx/puris/backend/stock/controller/StockController.java:73]: Javadoc comments should be added.

File: pom.xml (hunk 1 above)
Comment (tom-rm-meyer-ISST) [reply pom.xml]: Accidentially. Upgraded to 3.0.6

File: src/main/java/org/eclipse/tractusx/puris/backend/stock/controller/StockController.java (hunk 3 above)
Comment (tom-rm-meyer-ISST) [reply src/main/java/org/eclipse/tractusx/puris/backend/stock/controller/StockController.java:73]: Will be done in #31

**Review by mhellmeier — APPROVED** [review]
LGTM

**Pull request merged**